docs/source/whatsnew/1.0.0a.rst

```diff
@@ -0,0 +1,41 @@
+.. _whatsnew_100a:
```

Comment:
I really like having whatsnew displayed on readthedocs, rather than somewhat buried on GitHub.  Would be great to move this to pvlib also.

Reply:
I don't follow. pvlib's what's new is on readthedocs: https://pvlib-python.readthedocs.io/en/latest/whatsnew.html 

The only difference is that the pvlib PR template doesn't currently link to the `whatsnew` directory on its github page.

Reply:
So it is. Please disregard. The "somewhat buried on GitHub" comment applies to a different project's repository which I was just looking at.

Reply:
This did lead me to discover that the pvlib readthedocs "stable" build (the default when you go to `pvlib-python.readthedocs.io`) was for the short-lived `v0.6.4` tag rather than the current `v0.6.3` release. I fiddled with rtd's wipe and rebuild options to fix it.